test/cmocka/src/audio/selector/selector_test.c

```diff
@@ -129,7 +130,7 @@ static void verify_s16le_Xch_to_1ch(struct comp_dev *dev, struct comp_buffer *si
 	uint16_t source_in;
 	uint16_t destination;
 
-	for (i = 0; i < source->size / sizeof(uint16_t); i += in_channels) {
+	for (i = 0; i < source->size / sizeof(uint32_t); i += in_channels) {
```

Comment:
I don't see why this change is not separate or the meaning of this. Can you explain?

Reply:
Will approve once this is clarified, everything else looks good to me.

Reply:
bug, changed